pkg/lib/config/ui.go

```diff
@@ -37,8 +39,11 @@ type UIConfig struct {
 	DarkThemeDisabled      bool              `json:"dark_theme_disabled,omitempty"`
 	LightThemeDisabled     bool              `json:"light_theme_disabled,omitempty"`
 	WatermarkDisabled      bool              `json:"watermark_disabled,omitempty"`
+	DirectAccessDisabled   bool              `json:"direct_access_disabled,omitempty"`
 	// client_uri to use when client_id is absent.
 	DefaultClientURI string `json:"default_client_uri,omitempty"`
+	// branding_page_uri to use when client_id is absent.
+	DefaultBrandingPageURI string `json:"default_branding_page_uri,omitempty"`
```

Comment:
```
// brand_page_uri is shown when the UI has direct_access_disabled.
BrandPageURI string `json:"brand_page_uri,omitempty"`
```

That is, please rename this config to `brand_page_uri` and update the comment.

Reply:
renamed all related config name